modules/custom/openy_activity_finder/config/install/openy_activity_finder.settings.yml

```diff
@@ -1,6 +1,7 @@
 backend: openy_activity_finder.solr_backend
 index: default
 ages: "6,6mos\r\n12,12mos\r\n18,18mos\r\n24,2yrs\r\n36,3yrs\r\n48,4yrs\r\n60,5yrs\r\n72,6yrs\r\n84,7yrs\r\n96,8yrs\r\n108,9yrs\r\n120,10yrs\r\n132,11yrs\r\n144,12yrs\r\n156,13yrs\r\n168,14yrs\r\n180,15yrs\r\n192,16+yrs\r\n660,55+yrs"
+weeks: ""
```

Comment:
let's provide hook_update_N and update this particular parameter in already  existing configurations of installed sites

Reply:
Will it be enough: to check it is not set and update it only in that case?

Reply:
yes, it'll be enough @NightHunterSV

Reply:
Done. Check, please